lstchain/io/lstcontainers.py

```diff
@@ -51,23 +56,34 @@ class DL1ParametersContainer(Container):
     disp_miss = Field(None, 'disp_miss [m]', unit=u.m)
     src_x = Field(None, 'source x coordinate in camera frame', unit=u.m)
```

Comment:
`np.nan * u.m`

Reply:
Same as above. Trying to make minimal changes...